Convert HTML tables to Markdown tables

jabalazs opened the pull request:
I'm converting HTML tables to markdown following kramdown [specifications](https://kramdown.gettalong.org/syntax.html#tables), as suggested in #1.

I still haven't converted table 3 because the HTML version contains multi-line cells, which kramdown doesn't support. Should we keep the HTML version? or perhaps separate that table into 2 separate tables, one for dependency parsing, and another for constituency parsing?

jabalazs committed:
Convert Table 1 to markdown and fix wrong links

jabalazs committed:
Convert Table 2 to markdown

jabalazs committed:
Convert Table 4 to markdown

jabalazs committed:
Convert Table 5 to markdown

jabalazs committed:
Convert Table 6 to markdown

jabalazs committed:
Convert Table 7 to markdown

jabalazs committed:
Convert Table 8 to markdown

omarsar commented:
@jabalazs thanks for the PR! Are you planning to convert the other remaining tables? As for Table 3, we will think about how to best split it.

jabalazs committed:
Convert Table 9 to markdown

jabalazs commented:
I just finished converting the remaining tables!

As for table 3, just tell me whenever you make a decision and I'll convert it as well.

omarsar commented:
@jabalazs Awesome! Okay, you can go ahead and split table 3 into two tables (Dependency Parsing and Constituency Parsing) and you can include the descriptions below for the two tables. How about that?

Descriptions for Table 3
Table 3.1: Dependency Parsing (UAS/LAS = Unlabeled/labeled Attachment Score; WSJ = The Wall Street Journal Section of Penn Treebank)

Table 3.2: Constituency Parsing

jabalazs commented:
Sounds good! I'll split it now then.

jabalazs committed:
Split Table 3 into 2 markdown tables

- Move score descriptions to table headers
- Change table titles accordingly
- Change "F1 Score" to "F1 %" for consistency with the other tables
